doc/source/user_guide/text.rst

```diff
@@ -94,7 +94,12 @@ l. For ``StringDtype``, :ref:`string accessor methods<api.series.str>`
 2. Some string methods, like :meth:`Series.str.decode` are not available
    on ``StringArray`` because ``StringArray`` only holds strings, not
    bytes.
-
+3. In comparision operations, :class:`arrays.StringArray` and ``Series`` backed
+   by a ``StringArray`` will return an object with :class:`arrays.BooleanDtype`,
+   rather than a ``bool`` dtype object, depending on whether
+   there are missing values. Missing values in a ``StringArray`` will propagate
```

Comment:
is there a reason we are not *always* returning a BooleanArray, or am I mis-reading this? this would be *very* confusing if this is indeed the case.

Reply:
I think it is stated confusingly. When the dtype is "string" (not object), comparisons always return BooleanDtype. 
Tom, I think you can removed the ", depending on whether there are missing values"

Reply:
Fixed.